### ISSUE TITLE
Is OpenPose multi-batch inference tested ? 

### ISSUE THREAD

**Issue by TheBobbyliu** [zerollzeng/tensorrt-zoo#33]
I tried to apply multi-batch inference with OpenPose(fp16), but it failed to output any keypoint results.

**To Reproduce**
1. Set max_batch_size to 2.
2. Create an input_data that is double the size of the origin vector (for testing, I simply insert input_data into input_data's end iterator).
3. Do Inference.
4. Print the output.

**Result**
No keypoints are printed, and the mpPeaksCpu seems to be filled with zero.
(I've already tested the program when batch size is set to 1 and confirm the correctness of the keypoint outputs.)

**Comment by zerollzeng** [comment]
can it work correctly under fp32?

**Comment by zerollzeng** [comment]
@TheBobbyliu

**Comment by zerollzeng** [comment]
Closed due to inactivity

**Comment by drostifrosti** [comment]
I encountered the same behaviour, that no keypoints are generated for a second image, both with fp16 as well as fp32, although the mpPeaksCpu array does indeed have meaningful values greater 0 in the upper half.
In OpenPose::DoInference I did:
`op::connectBodyPartsCpu(poseKeypoints2, poseScores2, mpResizeMapCpu+mResizeMapSize/8, mpPeaksCpu+mPeaksSize/8,...`
with shifted starting adressses.
How can I approach this problem to also get keypoints for the second image?
